lib/Blin/Tester/Zef.rakumod

```diff
@@ -0,0 +1,72 @@
+unit class Blin::Tester::Zef;
+
+has $.path;
+has $.binary;
+has $.output-failed;
+has $!zef-config-path;
+has @.sources;
+
+
+
+submethod BUILD ( ) {
```

Comment:
Can't this be TWEAK?

Reply:
It can, it's just a habit to use `BUILD` since it gives more control over object construction. but in this case `TWAEK` makes more sense, since there is no arguments to `.new` anyway. I replaced `BUILD` with `TWEAK`.